src/components/_common/office-details/officeInformation/OfficeInformation.tsx

```diff
@@ -0,0 +1,69 @@
+import { Accordion, BodyShort, Heading } from '@navikt/ds-react';
+import { usePageConfig } from 'store/hooks/usePageConfig';
+import { translator } from 'translations';
+import { OfficeDetailsData } from 'types/content-props/office-details-props';
+import { formatAddress } from '../utils';
+
+import styles from './OfficeInformation.module.scss';
+
+export interface OfficeInformationProps {
+    officeData: OfficeDetailsData;
+}
+
+export const OfficeInformation = ({ officeData }: OfficeInformationProps) => {
+    const { language } = usePageConfig();
+    const getOfficeTranslations = translator('office', language);
+
+    const { postadresse, organisasjonsnummer, enhetNr } = officeData;
+
+    const visitingAdress = formatAddress(postadresse, true);
+    const postalAddress = formatAddress(postadresse, true);
+
```

Comment:
Trenger vi to av denne?

Reply:
Rart. Det burde ha vært `beliggenhet` på visitingAddress. Fikset! Samt skrivefeil.